runtime/browser/popup.cc

```diff
@@ -177,23 +177,19 @@ Popup* Popup::CreatePopup(NativeWindow* window) {
   elm_layout_theme_set(layout, "layout", "popup", kLayoutTheme);
```

Comment:
layout is not used if formfactor is not Circle, So these code also disable when formfactor is not circle